### ISSUE TITLE
Unable to compile with asp.DLL library 64 DLL project. But the 32-bit can compile.

### ISSUE THREAD

**Issue by woeoio** [twinbasic/twinbasic#1662]
**Describe the bug**
Unable to compile with asp. DLL library 64 DLL project. But the 32-bit can compile.

**To Reproduce**

**Expected behavior**

**Screenshots**
![image](https://github.com/twinbasic/twinbasic/assets/24835883/64cc9559-06e6-4046-ac2a-da69835c094d)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - twinBASIC compiler version ： 0.15.380

**Additional context**

**Comment by fafalone** [comment]
I don't believe ASPTypeLibrary has a 64bit version, have you found one? You can't just use the same one as vb6.

**Comment by woeoio** [comment]
> I don't believe ASPTypeLibrary has a 64bit version, have you found one? You can't just use the same one as vb6/tB 32bit.

![image](https://github.com/twinbasic/twinbasic/assets/24835883/64028760-fa25-4ea5-8407-6ae510ad8b27)
if you install 64bit iis and set asp extend, you well find 64bit of asp.dll file, look up picture.

**Comment by woeoio** [comment]
> I don't believe ASPTypeLibrary has a 64bit version, have you found one? You can't just use the same one as vb6/tB 32bit.

![image](https://github.com/twinbasic/twinbasic/assets/24835883/36920cba-b06c-4abb-9b06-69cc7bffba40)

and,  vbs host has 64bit version too,,,,,vba has too,  all  basic family has 64bit ,, but not vb6 only !! 狗日的巨硬。
